docs/website/generator-errors.md

```diff
@@ -212,6 +212,10 @@ This usually indicates a bug in Xamarin.iOS/Xamarin.Mac; please [file a bug repo
 
 ### <a name='BI1070'/>BI1070: The type '{type}' is trying to inline the property '{property}' from the protocols '{protocol1}' and '{protocol2}', but the inlined properties are of different types ('{property1}' is int, while '{property2}' is int).
 
+### <a name='BI1071'/>BI1071: The BindAs type for the member "{type}.{member}" must be an array when the member's type is an array. See src/tests/generator/ghissue6863_method.cs for example.
```

Comment:
This documentation ends up on our website, which means that the reference to `src/tests/generator/ghissue6864_method.cs` will not be very meaningful.

It would be much better to copy the example code into this document.

Reply:
Where can I access the website? I've added an example to the doc

Reply:
Actually the documentation for the generator isn't available on the web (yet!), but we're publishing documentation for other tools (such as mtouch: https://docs.microsoft.com/en-us/xamarin/ios/troubleshooting/mtouch-errors - the generator errors would one day show up similiarly)